### PR TITLE
PR: Add check compability

### LINKED ISSUE #26: Use check_compatibility to deactivate this plugin in Python 2
This will make it to depend on Spyder 3.2

### PR DESCRIPTION
**Pull request by rlaverde**
Fixes #26 

Deactivate plugin in python2, and make it depend of spyder3.2
(We will need to wait until spyder3.2 release to CI to work, or install spyder from git)

### CODE REVIEW

**Comment by ccordoba12** [comment]
@rlaverde, please take a look at the `circle.yml` file of spyder-notebook to see how to do this.

The ideas are:

1. You need to install Spyder first using conda or pip to bring Spyder dependencies, like this: `pip/conda install spyder`
2. Then you need to remove that Spyder version to install it from Github: `pip/conda uninstall spyder`.
3. You can't use `pip install git+https://...` because Spyder doesn't use setuptools, so that will fail. Instead you need these commands:

        mkdir spyder-source && cd spyder-source
        wget -q https://github.com/spyder-ide/spyder/archive/3.x.zip && unzip -q 3.x.zip
        cd spyder-3.x && python setup.py install > /dev/null

4. Finally you need to come back to your initial parent directory and install spyder-reports:

        cd ../../ && python setup.py install --single-version-externally-managed --record=record.txt > /dev/null

File: setup.py [hunk 1]
```diff
@@ -37,7 +37,7 @@ def get_description():
     return data
 
 
-REQUIREMENTS = ['spyder', 'pweave']
+REQUIREMENTS = ['spyder>=3.2', 'pweave']
```
Comment (ccordoba12) [inline setup.py]: This needs to be changed to `spyder>=3.2.0.dev0` to work with our latest 3.x branch.

File: setup.cfg [hunk 2]
```diff
@@ -1,5 +1,2 @@
-[bdist_wheel]
-universal=1
-
```
Comment (ccordoba12) [inline setup.cfg]: Good catch!

File: ci/install.sh [hunk 3]
```diff
@@ -14,11 +14,18 @@ source activate test
...
 fi
 
+mkdir spyder-source && cd spyder-source
+wget -q https://github.com/spyder-ide/spyder/archive/3.x.zip && unzip -q 3.x.zip
+cd spyder-3.x
+conda install --file requirements/requirements.txt
+python setup.py install > /dev/null
+
+# come back to your initial parent directory and install spyder-reports
 python setup.py install --single-version-externally-managed --record=record.txt > /dev/null
```
Comment (ccordoba12) [inline ci/install.sh]: You forgot to come back to the parent directory here, so you need to add

    cd ../../ && python setup.py install ...

File: ci/install.sh [hunk 4]
```diff
@@ -14,11 +14,18 @@ source activate test
...
     pip install -q pweave
 fi
 
+mkdir spyder-source && cd spyder-source
+wget -q https://github.com/spyder-ide/spyder/archive/3.x.zip && unzip -q 3.x.zip
+cd spyder-3.x
+conda install --file requirements/requirements.txt
+python setup.py install > /dev/null
+
+# come back to your initial parent directory and install spyder-reports
```
Comment (ccordoba12) [inline ci/install.sh]: come -> Come
your -> the

**Comment by rlaverde** [comment]
could I add `conda-forge` channel? spyder3.2 dependencies aren't available in `defaults`, and install/uninstall will install old dependencies

**Comment by ccordoba12** [comment]
What dependencies are not available?

**Comment by rlaverde** [comment]
> What dependencies are not available?

rope>=0.10.5

**Comment by rlaverde** [comment]
Merging, the coverage decreased because test files were being taken into account

**Pull request merged**